src/udev/udev-builtin-net_id.c

```diff
@@ -262,6 +262,24 @@ static bool is_pci_bridge(sd_device *dev) {
         return strneq(p + 2, "04", 2);
 }
 
+static int parse_slot(const char *s, int *ret) {
+        int r;
+
+        assert(s);
+        assert(ret);
+
+        if (naming_scheme_has(NAMING_SLOT_NONZERO) && s[0] == '0')
+                return -EINVAL;
+
+        r = safe_atoi(s, ret);
```

Comment:
Do not assign value to `ret` when this function fails. So, please do like the following.
```
int r, k;

r = safe_atoi(s, &k);
if (r < 0)
    return r;
if (k <= 0)
    return -ERANGE;

*ret = k;
return 0;
```

Reply:
Thanks, will do.